doc/versionHistory.rst

```diff
@@ -5,6 +5,12 @@
 ##################
 Version History
 ##################
+-------------
+1.5.7
```

Comment:
I think since you updated the command line structure with new functionality this should be bumped to 1.6.0.